### DISCUSSION TITLE
Anyone using Teslemetry for Authentication?

### DISCUSSION THREAD

**Discussion by noobey** [alandtse/tesla#924] [General]
It looks like it does all the hard work of setting up the Fleet API for you:
https://teslemetry.com/

I'm not sure how much confidence I have in the security of it though, I may also be over paranoid picking up on spelling errors on the site.

**Comment by fhteagle** [comment]
Just started checking this out as an interim solution. I agree that it would be nice if this integration could use teslemetry.com as the Fleet API proxy for those that would prefer to go that route.
